diagnostics/convective_transition_diag/convecTransCriticalCollapse_util.py

```diff
@@ -618,7 +619,8 @@ def convecTransCriticalCollapse_plot(argsv1,argsv2,argsv3,argsv4,argsv5,argsv6):
     # create figure canvas
     fig_cts = mp.figure(figsize=(figsize1,figsize2))
     
-    fig_cts.text(s='Convective Transition Collapsed Statistics ('+MODEL+')', x=0.5, y=1.02, ha='center', va='bottom', transform=fig_cts.transFigure, fontsize=16)
+    #fig_cts.text(s='Convective Transition Collapsed Statistics ('+MODEL+')', x=0.5, y=1.02, ha='center', va='bottom', transform=fig_cts.transFigure, fontsize=16)
```

Comment:
Is there a need to keep the old line around?

Reply:
Deleted.